lib/ansible/modules/cloud/cloudstack/cs_nic.py

```diff
@@ -205,14 +214,14 @@ def get_secondary_ip(self):
 
     def present_nic(self):
         nic = self.get_nic()
-        if not self.get_secondary_ip():
+        if self.vm_guest_ip and not self.get_secondary_ip():
```

Comment:
Could you give me some more infos about this change? It seems to me this will break some functionality, because `vm_guest_ip` is maked to be optional but this condition makes it a required param, right?